Godot html5 sample (#354)
dfinity/examples: hosting/unity-webgl-template/README.md

$ dfx deploy
```

> If you get 500 error after deploying to the IC mainnet, try to use `raw` keyword in the URL like https://\<canister-id\>.raw.ic0.app. Unity generates pretty large `.data` and `.wasm` files which can only be served using `raw` keyword.

Reviewer: ```suggestion
If you get error code 500 after deploying to the IC mainnet, try to use `raw` keyword in the URL like this: `https://\<canister-id\>.raw.ic0.app`.
```
same reasoning as above
